### PR TITLE
Dcj/213269 - added language picker

### PR DESCRIPTION
**Pull request by dpanta-plenty**

### CODE REVIEW

**Review by sene-plenty — APPROVED** [review]

File: src/lang/de/Manual.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+selectLanguage = "Sprache wählen"
+thisLang = "Deutsch"
+otherLang = "English"
```
Comment (ksted) [inline src/lang/de/Manual.properties:3]: Do we need these? The keys don't seem to be used anywhere.

File: src/lang/en/Manual.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+selectLanguage = "Select language"
+thisLang = "English"
+otherLang = "German"
```
Comment (ksted) [inline src/lang/en/Manual.properties:3]: See above.

File: src/partials/header-content.hbs [hunk 3]
```diff
@@ -21,7 +21,19 @@
                     </div>
                     <a class="menu-item" href="/en-gb/plentydevtool/main/plentydevtool-introduction.html">plentyDevTool</a>
                     <a class="menu-item" href="https://knowledge.plentymarkets.com/en" target="_blank">User manual</a>
-                </div>
+                    <div class="dropdown menu-item">
```
Comment (ksted) [inline src/partials/header-content.hbs:24]: If we want to use only one header-content.hbs, we have to add a conditional that the element doesn't show up on developers.plentymarkets.com.

**Review by ksted — CHANGES_REQUESTED** [review]

File: src/partials/header-content.hbs (hunk 3 above)
Comment (pfrincu-plenty) [reply src/partials/header-content.hbs:25]: check if its correct!

Comment (ksted) [reply src/partials/header-content.hbs:25]: It looks right to me at least, yeah.

**Review by ksted — APPROVED** [review]

**Pull request merged**